app/models/concerns/courses/edit_options/subjects_concern.rb

```diff
@@ -13,6 +13,15 @@ def potential_subjects
             class: CourseSerializersService.new.execute,
           )[:data]
         end
+
+        def potential_modern_languages
```

Comment:
shouldn't it be just be ` modern_languages_subject`

Reply:
I feel like this is more descriptive of its function

Reply:
https://github.com/DFE-Digital/manage-courses-backend/pull/926#discussion_r338504345